nss-day-cohort-29/reactive-nutshell-springwynne
# Rr friends

# Description

Complete friends functionality.  Builds friends array in state consisting of the actual user objects based on the connections in the "friends" endpoint.  Filters the news and events to only those of the current user and their friends.

This branch also include all of the events functionality (add, edit, delete)

## Type of change

- [x ] New feature (non-breaking change which adds functionality)


# Testing Instructions for Change Made

Checkout rr-friends
Run json-server and npm start
All functions on friends and events (add, edit, delete) should work

# Checklist:

- [x ] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [x ] I have commented my code, particularly in hard-to-understand areas
- [x ] My changes generate no new warnings
- [x ] I have added test instructions that prove my fix is effective or that my feature works`
